mapo/models/dynamics/gaussian_constant_stddev.py

```diff
@@ -81,7 +82,11 @@ def call(self, inputs, training=None, mask=None):
         """
         # pylint: disable=unused-argument,attribute-defined-outside-init
         obs, actions = inputs
-        obs = self.obs_layer(obs)
+        # obs = self.obs_layer(obs)
+
+        if isinstance(obs, (tuple, list)):
+            obs, _ = obs
```

Comment:
In a typical MDP formulation the dynamics model should be stationary (i.e., it should not depend on the value of the timestep).

I didn't use the `TimeConcatObservationLayer` here exactly because of that...

Reply:
Ah, I'm stupid, I didn't realize I was looking at the dynamics model